rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1133,6 +1133,8 @@ private void processPegoutsInBatch(
             BtcTransaction generatedTransaction = result.get().getBtcTx();
             // TODO: Update to call addPegoutTxToReleaseTransactionSet with the RskHash that calls the updateCollections
             releaseTransactionSet.add(generatedTransaction, rskExecutionBlock.getNumber());
+            Keccak256 rskTxHash = releaseRequestQueue.getEntries().get(0).getRskTxHash();
+            eventLogger.logBatchPegoutCreated(generatedTransaction, rskTxHash.getBytes());
```

Comment:
We want to log all the pegout requests included in the batch

Reply:
Removed Those Lines, It Will Be Done In The New Ticket